### DISCUSSION TITLE
Which Raspberry Pi model is the current suggested version to get for performance etc?

### DISCUSSION THREAD

**Discussion by Billium28** [mcguirepr89/BirdNET-Pi#1151] [General]
I had my first year of Birdnetpi running last year to great success. It took a few microphones to get a good response.. I finally setllted on this [AOM-5024L-HD-R](https://www.mouser.com/ProductDetail/PUI-Audio/AOM-5024L-HD-R?qs=j%252B1pi9TdxUZP%252BNk1SD4XMw%3D%3D&countryCode=US&currencyCode=USD) . But since the Fall came to a close I have used my Rpi on a new project and want to buy a new one to start a permanent Birdnetpi.

I used a 3b and I think it worked ok but I want to get a Pi 4 for future proofing. My main question is how much memory do I need for this? 1-8gb is a spread I want to get right the first time..

**Comment by tphakala** [comment]
I don't know if you have noticed or not but there is not much of a future for BirdNET-Pi, development has been stagnant for more than a year now. If you are happy for performance of RPI 3b I'd stick with it.

**Reply by rawcomposition** [reply]
In terms of development being stagnant, I recently came across this GO based version which looks great https://github.com/tphakala/birdnet-go
